src/main/java/org/gelecekbilimde/scienceplatform/common/model/response/GenericResponse.java

```diff
@@ -0,0 +1,38 @@
+package org.gelecekbilimde.scienceplatform.common.model.response;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Getter;
+import org.gelecekbilimde.scienceplatform.common.util.RandomUtil;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Builder
+public class GenericResponse<T> {
```

Comment:
Hocam burada biz aslında success response için bir sınıf tanımlıyoruz. Error response için ismi ErrorResponse diye tanımladık. Burda da SuccessGenericResponse veya GenericSuccessResponse olması daha iyi olmaz mı ilk görüşte şahis olarak genel bir generic ( hem success hem de error için ) bir sınıf sandım.